src/ledger/LedgerManagerImpl.cpp

```diff
@@ -782,13 +785,20 @@ LedgerManagerImpl::closeLedger(LedgerCloseData const& ledgerData)
 
         CLOG(ERROR, "Ledger")
             << "Full LCL: " << xdr::xdr_to_string(getLastClosedLedgerHeader());
+        CLOG(ERROR, "Ledger") << POSSIBLY_CORRUPTED_QUORUM_SET;
```

Comment:
this is wrong: bad hash is likely because local database or buckets were corrupted